worker/uniter/runner/debug/client.go

```diff
@@ -15,12 +15,12 @@ type hookArgs struct {
 }
 
 // ClientScript returns a bash script suitable for executing
-// on the unit system to intercept hooks via tmux shell.
-func ClientScript(c *HooksContext, hooks []string) string {
-	// If any hook is "*", then the client is interested in all.
-	for _, hook := range hooks {
-		if hook == "*" {
-			hooks = nil
+// on the unit system to intercept hooks or actions via tmux shell.
+func ClientScript(c *HooksContext, args []string) string {
```

Comment:
args doesn't mean much. hooksOrActions is ugly but at least meaningful. or just call it "match", and extend doc comment?